### ISSUE TITLE
Images missing alt text

### ISSUE THREAD

**Issue by nathakits** [pupil-labs/pupil-docs#142]
there are two images missing alt text at `users-docs` -> `pupil-capture` -> `before every calibration` section

**Comment by nathakits** [comment]
@willpatera - missing alt text PR https://github.com/pupil-labs/pupil-docs/pull/141

**Comment by nathakits** [comment]
Merged. Closing.
